alvr/client_core/src/lib.rs

```diff
@@ -273,6 +274,32 @@ impl ClientCoreContext {
             }
         }
 
+        // send_tracking() expects hand data in the multimodal protocol. In case multimodal protocol
```

Comment:
i think it's implicit

Reply:
eeeh, if you're gonna put a comment, add all relevant info into the comment
(the comment just threw me a little off of the fact it's for hand tracking dunno)

Reply:
No i think you misunderstand. Here I wrote: "In case multimodal protocol is not supported, convert back to legacy protocol". I did not write: "In case multimodal protocol is not supported, push controller motions". by saying "convert back to legacy protocol" I mean do the necessary checks ("if we're using hand tracking") to convert to legacy protocol.

Reply:
I just applied to comment to the action path of the if, you apply it to either case, you're right